packages/devtools-source-map/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "devtools-source-map",
-  "version": "0.16.0",
+  "version": "0.16.1",
```

Comment:
I think the failures are because you bumped this, but not the version in the repo-root `package.json` FYI.

Probably easiest to just leave both as-is?